Bekräfta-modal för avsluta partnerskap + lite till

jobjorn opened the pull request:
* lite nya färger
* justering av Submit-knappen
* bekräfta-modal för avsluta partnerskap

jobjorn committed:
utveckling av partnership-formuläret

jobjorn committed:
uppdaterar prisma

jobjorn committed:
uppdaterar lite paket

jobjorn committed:
gör färdigt partner-kopplingsrutan

jobjorn committed:
Korrigerat enligt kommentarer på PR1

jobjorn committed:
föreslår default för font-size istället

jobjorn committed:
Merge branch 'main' into overview

jobjorn committed:
kompletterande färger

jobjorn committed:
color-funktionalitet till submit-knappen

jobjorn committed:
bekräfta-modal till avsluta partnerskap

jobjorn committed:
ändra till accepted=true vid acceptering

jobjorn committed:
formuleringsändring

HedvigM commented on components/PartnerForm.tsx:
```diff
@@ -30,12 +39,22 @@ export const PartnerForm: React.FC<{ user: UserWithPartners | null }> = ({
 
+  const [cancelModalOpen, setCancelModalOpen] = useState<boolean>(false);
+  const handleCloseCancelModal = () => {
+    console.log('close cancel modal!');
```
remove console.log

jobjorn replied:
ordnat

HedvigM commented on components/PartnerForm.tsx:
```diff
@@ -30,12 +39,22 @@ export const PartnerForm: React.FC<{ user: UserWithPartners | null }> = ({
+    setCancelModalOpen(false);
+  };
+  const handleOpenCancelModal = () => {
+    console.log('open cancel modal!');
```
here are one as well

HedvigM commented on components/PartnerForm.tsx:
```diff
@@ -30,12 +39,22 @@ export const PartnerForm: React.FC<{ user: UserWithPartners | null }> = ({
     null
   );
 
+  const [cancelModalOpen, setCancelModalOpen] = useState<boolean>(false);
```
Det hade varit nice om modalen stängde sig själv efter att man klickat på "jag är säker på att jag vill avsluta partnerskap" och att förändringen genast syns på översiktssidan. När jag testade var jag tvungen att ladda om sidan för att det skulle synas, kanske en "optimistic update" för detta?

jobjorn replied:
detta är löst genom fixad caching, hoppas jag

HedvigM commented on components/PartnerForm.tsx:
```diff
@@ -120,15 +139,14 @@ export const PartnerForm: React.FC<{ user: UserWithPartners | null }> = ({
           </p>
           <Submit name="invitation" value="accept">
-            Acceptera
+            Ja
```
bra förändring, känns mer avslappnat

jobjorn replied:
tack älskling

HedvigM commented on components/PartnerForm.tsx:
```diff
@@ -157,7 +215,15 @@ export const PartnerForm: React.FC<{ user: UserWithPartners | null }> = ({
+          <Button
+            variant="contained"
+            onClick={() => {
+              console.log('knapptryck');
```
ännu en console.log

HedvigM reviewed (commented):
Lite småsaker bara, snyggt implementerat! Jag tycker dock att vi skall få till en optimistic update innan vi mergar in i main.

jobjorn committed:
minus console.log

jobjorn committed:
rättar vilken tag som ska revalidatas

jobjorn committed:
modalen stängs om allting gick bra

jobjorn committed:
typar action-functions och lägger till tid i StatusMessage

jobjorn committed:
visa endast det senaste statusmessaget om det finns flera

det finns flera olika formulär på sidan och de har alla varsin statusMessage-state, som sen kombineras

HedvigM committed:
remove a console.log

jobjorn committed:
förbereder merge

jobjorn committed:
Merge branch 'main' into overview
